fem/lininteg_domain.cpp

```diff
@@ -260,7 +229,7 @@ static void DLFEvalAssemble(const FiniteElementSpace &fes,
    const int ne = fes.GetMesh()->GetNE();
    const int *M = markers.Read();
    const double *B = maps.B.Read();
-   const double *J = geom->J.Read();
+   const double *J = geom->detJ.Read();
```

Comment:
maybe we name this `detJ` as well, just for consistency?

Reply:
Fixed in 1f18293a2f55152c15153084045d9b0795079870